src/app/registration/registration.component.css

```diff
@@ -2,17 +2,21 @@ a {
   color: #459fed;
   text-decoration: none;
 }
+
 .title1 {
```

Comment:
title1 and title2 is not good naming. Use BEM naming convention, also try to make naming more descriptive, try to think what content will be in the title, etc.